python/lsst/pipe/analysis/analysis.py

```diff
@@ -1096,10 +1096,12 @@ def plotQuiver(self, catalog, description, plotInfoDict, areaDict, log, cmap=plt
         yield Struct(fig=fig, description=description, stats=stats, statsHigh=None, dpi=150, style="quiver")
 
     def plotRhoStatistics(self, description, plotInfoDict, log, treecorrParams, stats,
-                          zpLabel=None, forcedStr=None, postFix="", flagsCat=None, uberCalLabel=None):
+                          zpLabel=None, forcedStr=None, postFix="", flagsCat=None, uberCalLabel=None,
+                          verifyJob=None):
         """ Plot Rho Statistics """
```

Comment:
Should be (with appropriate level of indentation):
"""Plot Rho Statistics.
"""